### DISCUSSION TITLE
Не работает Discord / Discord doesn't work

### DISCUSSION THREAD

**Discussion by keksik42** [Flowseal/zapret-discord-youtube#1053] [Q&A]
Не запускается дискорд когда программа запущена, хотя другие сайты работают нормально
Discord does not start when the program is running, although other sites work fine

**Comment by Flowseal** [comment]
Какие другие сайты?
https://github.com/Flowseal/zapret-discord-youtube/discussions/252

**Reply by keksik42** [reply]
Все остальные сайты указанные в list-general

**Reply by Flowseal** [reply]
Выполните советы в этой теме https://github.com/Flowseal/zapret-discord-youtube/discussions/252
И пробуйте другие батники

**Comment by Borsh01** [comment]
![image](https://github.com/user-attachments/assets/90459086-8755-4385-b96b-c0cd2672510a)
как пофиксить??

**Reply by IndeecFOX** [reply]
@Borsh01 очевидно, как и всегда с архивами - не запускать в архиве, а распаковать в папку) Желательно без пробелов и кириллицы в пути)

**Comment by Vladislavik-rus** [comment]
У меня такая же ситуация. Discord перестал запускаться с помощи zapret (1.6.1) и постоянно зависает на проверке обновлений, но он (zapret) продолжает помогать с обходом YouTube

**Reply by Flowseal** [reply]
https://github.com/Flowseal/zapret-discord-youtube/discussions/252
